emily/cdk/package.json

```diff
@@ -5,16 +5,15 @@
     "emily": "bin/emily.js"
   },
   "scripts": {
-    "build": "(cd ../api-definition && npm run build-openapi) && tsc",
+    "synth": "tsc && npx aws-cdk synth -q --no-staging",
```

Comment:
The original `"build"` did something a little misleading.

It would:
1. make the openapi template from the `api-description` sibling directory
1. Compile the typescript to javascript

That just builds the sources, but it doesn't build any `cdk` artifact. Normally you'd run `npm run build` and then run some command like `cdk deploy` or `cdk synth` that utilize the generated javascript.`cdk deploy` synthesizes the template and then deploys the artifacts to AWS, and the `cdk synth` command just makes the template.

I've changed this so that it'll take the `typescript` sources and turn them into a the cdk template right off the bat and assumes that the openapi template has already been generated. This is a better separation of responsibilities given the new structure of the Makefile. In the build command before there was no usable output (no template) and it modified sources even when it didn't need to be rebuilt.

I also changed the name to `synth` because it better matches the cdk idioms.

Reply:
I see. Thank you for the explanation, I did not know that.

So going forward, when we do `make build` it would not build the cdk template files anymore. That would get done if we run integration tests, setup a dev env (but with local configuration), or when/if we add some commands to deploy.